provider/app.py

```diff
@@ -114,7 +115,17 @@ def healthRoute():
         consumerInfo[consumerId] = {'secondsSinceLastPoll': lastPollDelta.total_seconds()}
         consumerReports.append(consumerInfo)
 
+    health = Health()
+    healthReport['uptime'] = '%d seconds' % uptimeSeconds
```

Comment:
Is this now redundant with `boot_time`? If not, is there some other metric that `psutil` provides that could replace this?

If so, then this whole thing could be shortened to something like `return jsonify(Health.generateReport())`